src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.page {
+  min-height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
page is a block so use separate style file for it

Reply:
Thanks, didn't notice that